### PR TITLE
Update displayName in Microsoft Graph TOC

### PR DESCRIPTION
**Pull request by Linda-Editor**
PROBLEM: The displayName tag was attached to folders, but it doesn't work on empty folders. Therefore, users could not get to the Overview topics for many sections when using the search filter. 

FIXES:
- Moved displayName to first href reference in a category; usually, this was the Overview topic
- Added missing displayName elements where needed
- Updated some of the Overview topic TOC titles to be more descriptive so as to provide a better search experience. For example, in the Use the APIs section, there were several Overview topics that would come up in the same search, and they all had the same search result: Overview - Develop > Use the API. So it was impossible to tell what they were the overview for. Adding a more descriptive title fixed that issue.
- Made other tweaks where needed

### CODE REVIEW

File: concepts/toc.yml [hunk 1]
```diff
@@ -12,16 +12,18 @@ items:
       - name: Services in Microsoft Graph
         href: overview-major-services.md
       - name: What's new
-        expanded: true
+        expanded: false
```
Comment (Lauragra) [inline concepts/toc.yml:15]: Please don't collapse any nodes for now because specific nodes are expanded by design.

File: concepts/toc.yml [hunk 2]
```diff
@@ -48,6 +51,8 @@ items:
             href: /graph/tutorials/python
           - name: TypeScript
             href: /graph/tutorials/typescript
+      - name: Quick start
```
Comment (Lauragra) [inline concepts/toc.yml:54]: Maybe we should move the Quick start to come right before the Tutorials node, just because this might be the "first" thing they do before completing a tutorial walkthrough?

File: concepts/toc.yml [hunk 3]
```diff
@@ -75,17 +82,19 @@ items:
...
+            displayName: applications, apps
           - name: Automate SAML-based SSO configuration
             href: application-saml-sso-configure-api.md
           - name: Configure application proxy
             href: application-proxy-configure-api.md
       - name: Accessing data at scale
-        displayName: Microsoft Graph Data Connect
         items:
-          - name: Overview of Data Connect
+          - name: Data Connect overview
```
Comment (Lauragra) [inline concepts/toc.yml:92]: For consistency with the other nodes, I suggest we make the TOC entry "Overview".

File: concepts/toc.yml [hunk 4]
```diff
@@ -649,13 +679,14 @@ items:
                 href: migrate-azure-ad-graph-deploy-test-extend.md
               - name: Migration FAQ
                 href: migrate-azure-ad-graph-faq.md
+                displayName: migrate, Azure AD Graph, frequently asked questions
```
Comment (Lauragra) [inline concepts/toc.yml:682]: For the other FAQ topic we have FAQ in the displayName; is it better to use the same one consistently?

File: concepts/toc.yml [hunk 5]
```diff
@@ -96,11 +105,12 @@ items:
             href: data-connect-pam.md
           - name: Frequently asked questions
```
Comment (Lauragra) [inline concepts/toc.yml:106]: We should probably change this entry here to FAQ, and update it in the topic title as well. FAQ is a recognized acronym and is a much more commonly searched term.

File: concepts/toc.yml [hunk 6]
```diff
@@ -152,16 +163,19 @@ items:
       - name: Compliance
         href: compliance-concept-overview.md
       - name: Connecting external content
-        displayName: connectors
         items:
-          - name: Overview of Microsoft Graph connectors
+          - name: Connectors overview
```
Comment (Lauragra) [inline concepts/toc.yml:167]: This is another place where, for consistency, we might just use Overview.

File: concepts/toc.yml [hunk 7]
```diff
@@ -173,30 +187,35 @@ items:
...
           - name: Business rules
             href: bookingsbusiness-business-rules.md
+            displayName: Bookings, Microsoft Bookings, booking
       - name: Device and app management
         expanded: true
         items:
           - name: Cloud printing
             items:
-              - name: Overview
+              - name: Cloud printing overview
```
Comment (Lauragra) [inline concepts/toc.yml:216]: Not sure why the change in some places to include the node name in the TOC entry while in other nodes we just use Overview. Let's make all the nodes consistent and just use Overview for the overview topic in each node.

File: concepts/toc.yml [hunk 8]
```diff
@@ -350,14 +374,13 @@ items:
               - name: Error codes
                 href: onenote-error-codes.md
       - name: Notifications (deprecated)
-        displayName: Notifications
         items:
           - name: Overview
             href: notifications-concept-overview.md
+            displayName: deprecated
```
Comment (Lauragra) [inline concepts/toc.yml:380]: Are we sure that deprecated is what we want for the displayName? Maybe we include Notifications and deprecated? It's possible that even though the API is deprecated, someone will still search for it in the filter.

File: concepts/toc.yml [hunk 9]
```diff
@@ -350,14 +374,13 @@ items:
...
           - name: Get started
-            displayName: Notifications
             items:
-              - name: Overview
+              - name: Get started overview
```
Comment (Lauragra) [inline concepts/toc.yml:383]: This might be a better TOC title: Integrate with notifications

File: concepts/toc.yml [hunk 10]
```diff
@@ -567,8 +596,9 @@ items:
                 href: universal-print-webhook-notifications.md
               - name: Change notifications for Microsoft Teams
                 items:
-                  - name: Overview
+                  - name: Change notifications for Microsoft Teams resources
```
Comment (Lauragra) [inline concepts/toc.yml:599]: Seems like it's better to keep this as Overview for readability/scannability since it basically duplicates that parent node title.

File: concepts/toc.yml [hunk 11]
```diff
@@ -595,10 +625,10 @@ items:
               - name: Get changes to users (example)
                 href: delta-query-users.md
           - name: Add custom data
-            displayName: Outlook calendar, Groups, Outlook mail, Outlook contacts, Outlook contact, People, Users
             items:
-              - name: Overview
+              - name: Add custom data to resources using extensions
```
Comment (Lauragra) [inline concepts/toc.yml:629]: I think we can keep this as Overview.

File: concepts/toc.yml [hunk 12]
```diff
@@ -787,17 +824,22 @@ items:
               - name: Right-to-left support
                 href: toolkit/customize-components/right-to-left.md
       - name: Resources
-        expanded: true
+        expanded: false
```
Comment (Lauragra) [inline concepts/toc.yml:827]: We should keep everything expanded for now.

File: concepts/toc.yml (hunk 2 above)
Comment (Linda-Editor) [reply concepts/toc.yml:54]: Done.

File: concepts/toc.yml (hunk 3 above)
Comment (Linda-Editor) [reply concepts/toc.yml:92]: I had thought about that too, but the difference between the Data Connect and Connectors nodes and other overview nodes is that the folders don't say Data Connect or Connectors; they say "accessing data at scale" and "connecting external content".  And both of these were spelled out previously, probably by design. So that's why I didn't change them, and that's why I feel like we shouldn't change them.

File: concepts/toc.yml (hunk 4 above)
Comment (Linda-Editor) [reply concepts/toc.yml:682]: I thought about that too. It could go either way. But since you noticed it too, I'll go ahead and make them consistent.

File: concepts/toc.yml (hunk 6 above)
Comment (Linda-Editor) [reply concepts/toc.yml:167]: See my reply for Data Connect.

File: concepts/toc.yml (hunk 7 above)
Comment (Linda-Editor) [reply concepts/toc.yml:216]: There is a method to my madness, and I'm going to push back on this suggestion. There are multiple instances where the search results show the exact same wording because of multiple overview topics in the same top node. The only way the user can know which topic they're looking at is if we spell it out. One reason this is happening is because someone in the past has added the same displayName entries for multiple overview topics, mostly in the API section. But it also happens because in a given folder, there is more than one overview, but the search results don't differentiate. While it may make the TOC inconsistent, it improves the search functionality. And my guess is that users use the search function more than they browse through the TOC.

File: concepts/toc.yml (hunk 9 above)
Comment (Linda-Editor) [reply concepts/toc.yml:383]: Done

File: concepts/toc.yml (hunk 11 above)
Comment (Linda-Editor) [reply concepts/toc.yml:629]: I'm pushing back for the same reasons I mentioned before.

File: concepts/toc.yml (hunk 10 above)
Comment (Linda-Editor) [reply concepts/toc.yml:599]: For example, if you search on **Change notifications**, there are 6 results from the Develop > Use the API section. If we don't change the display name to be more specific, most of these results show up as Overview - Develop > Use the API. There's no way for the user to know which topic they're looking at, other than to painstakingly click each one, then go back and filter again, etc. And it also makes it look like the same result is showing up 6 times. I'm optimizing for the search experience, not the browse experience.

File: concepts/toc.yml (hunk 8 above)
Comment (Linda-Editor) [reply concepts/toc.yml]: Done

File: concepts/toc.yml (hunk 10 above)
Comment (Lauragra) [reply concepts/toc.yml:599]: Am I misunderstanding the purpose of the displayName attribute? If we apply a unique and specific displayName attribute to a TOC entry, won't that come up in the search filter regardless of the name of the TOC label?

Comment (Linda-Editor) [reply concepts/toc.yml:599]: Yes. However, under Use the API, where there are multiple Overviews, there are also multiple displayName entries for each TOC overview, meaning that all of the overview topics under Develop > API might show up in a single keyword search (such as Users). If we don't change the title for these overviews to be more specific, they show up identically in the search result as Overview - Develop > API. If you don't want this is to happen, we need to scope the keywords for each of these overviews to not include all the product/service names they're connected to. Getting identical search results for a given keyword is not helpful.

File: concepts/toc.yml [hunk 13]
```diff
@@ -94,13 +105,14 @@ items:
             href: data-connect-filtering.md
           - name: Integrate with PAM
             href: data-connect-pam.md
-          - name: Frequently asked questions
+          - name: Data Connect FAQ
```
Comment (Lauragra) [inline concepts/toc.yml:108]: The context is probably clear here; can this just be FAQ?

File: concepts/toc.yml [hunk 14]
```diff
@@ -173,30 +189,35 @@ items:
...
           - name: Business rules
             href: bookingsbusiness-business-rules.md
+            displayName: Bookings, Microsoft Bookings, booking
       - name: Device and app management
         expanded: true
         items:
           - name: Cloud printing
             items:
-              - name: Overview
+              - name: Cloud printing overview
```
Comment (Lauragra) [inline concepts/toc.yml:218]: I still recommend that we keep this node as Overview, to improve the browsing experience.

File: concepts/toc.yml [hunk 15]
```diff
@@ -208,11 +229,11 @@ items:
             displayName: Intune
           - name: Cloud PC (preview)
             href: cloudpc-concept-overview.md
-          - name: Device updates (preview)
-            displayName: Windows updates
+          - name: Device updates (preview)           
             items:
-              - name: Overview
+              - name: Device updates overview
```
Comment (Lauragra) [inline concepts/toc.yml:234]: Overview here, too.

File: concepts/toc.yml [hunk 16]
```diff
@@ -248,40 +270,44 @@ items:
...
                 href: tutorial-riskdetection-api.md
           - name: Governance
             items:
               - name: Manage access to resources
                 href: tutorial-access-package-api.md
+                displayName: governance, identity
               - name: Review access to your resources
                 items:
-                  - name: Overview of the access reviews API
+                  - name: Access reviews API overview
```
Comment (Lauragra) [inline concepts/toc.yml:308]: Let's make this just Overview.

File: concepts/toc.yml [hunk 17]
```diff
@@ -684,24 +719,26 @@ items:
...
       - name: Use the toolkit
         items:
-          - name: Toolkit overview
+          - name: Overview
             href: toolkit/overview.md
+            displayName: Microsoft Graph Toolkit, toolkit
           - name: Get started
             items:
-              - name: Overview
+              - name: Get started overview
```
Comment (Lauragra) [inline concepts/toc.yml:739]: Let's just make this "Get started".

File: concepts/toc.yml (hunk 13 above)
Comment (Linda-Editor) [reply concepts/toc.yml:108]: Only if we change the folder name to Data Connect. It's currently Accessing data at scale. Someone decided before my time that instead of using Data Connect for the folder name, we should put the intent here, which is fine. But we've received feedback that no one can find Data Connect resources.  Everything that comes up in search for this folder has the path **Learn > Accessing data at scale**.  People might not know that this means Data Connect. So I'd prefer to keep it for visibility.

File: concepts/toc.yml (hunk 14 above)
Comment (Linda-Editor) [reply concepts/toc.yml:218]: There are two overviews under Device and app management, so if they appear in search results together, the user has no way of knowing which is which. This is why I made this change.

Improving the browse experience = diminishing the search experience (in some cases). I'm trying to fix the search experience; your priority is the browse experience. It all depends on which you think the user will use more. If most of our users get to our site via Google search, might they not also search the TOC rather than browse it? I know that when I've wanted to find something in this TOC, I've always used the search filter because there are a lot of files in this docset. 

I'll make these changes because you insist on them even though I've tried to explain the method to my madness. But I don't agree with them.

File: concepts/toc.yml (hunk 16 above)
Comment (Linda-Editor) [reply concepts/toc.yml:308]: Done, but this also could be Access reviews API.

File: concepts/toc.yml (hunk 17 above)
Comment (Linda-Editor) [reply concepts/toc.yml:739]: Done
